modules/security-context-constraints-psa-opting.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-opting_{context}"]
+= Disabling pod security admission synchronization
+
+You can disable automatic pod security admission synchronization for a namespace.
+
+.Procedure
+
+* For each namespace for which you want to disable sychronization, change the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `false`. 
```

Comment:
I'd rather use "set" instead of "change" because the label is unlikely to be set.

Reply:
```suggestion
* For each namespace for which you want to disable sychronization, set the value of the `security.openshift.io/scc.podSecurityLabelSync` label to `false`. 
```